code/modules/mob/living/living_defines.dm

```diff
@@ -84,3 +84,9 @@
 
 	spawn_frequency = 10
 	bad_type = /mob/living
+
+	var/list/os_components = list(/obj/item/stock_parts/capacitor/one_star,
+								/obj/item/stock_parts/scanning_module/one_star,
+								/obj/item/stock_parts/manipulator/one_star,
+								/obj/item/stock_parts/micro_laser/one_star,
+								/obj/item/stock_parts/matter_bin/one_star) //One Star Robot Drop
```

Comment:
```suggestion
```
Put this list inside the procs. Otherwise, every living mob will have these stored on it. It might be better to make a #define somewhere, but the suggestions are good enough for now.